examples/appliance_device_snmp_v3_trap_destinations.py

```diff
@@ -30,7 +30,7 @@
 options = {
     "type": "Destination",
     "destinationAddress": "1.1.1.1",
-    "userId": "a5c9a94d-ce98-42bd-8f3a-b1856d6071b0",
+    "userId": "",
```

Comment:
You can remove this assignment as you are assigning the value down

Reply:
I have added empty userId key in the payload just to make people understand that "userId" is an important attribute in this payload.